src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, $cardWidth);
+  gap: $gapping;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, $cardWidth);
+  }
+
+  @media (min-width: 768px) {
+    grid-template-columns: repeat(3, $cardWidth);
+  }
+  
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $cardWidth);
+
+  }
```

Comment:
You can also try to set auto-fill to grid-template-columns and set max-width to container instead of using media-queries